README.md

```diff
@@ -47,6 +47,26 @@ npm test
 
 For detailed explanation on how things work, checkout the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
 
+## Run with Docker
+
+At a later stage we will make Docker images available via DockerHub. For now
+you can build/run as follows:
+
+``` bash
+docker build -t meggsimum/wegue:latest .
+
+docker run -it -p 8080:80 meggsimum/wegue:latest
+
+# in browser: open http:/127.0.0.1:8080
```

Comment:
Missing second `/` in URL. I suggest the following for the doc block:

```suggestion
``` bash
docker build -t meggsimum/wegue:latest .

docker run -it -p 8080:80 meggsimum/wegue:latest

Open
  - http://127.0.0.1:8080/ or
  - http://localhost:8080/?appCtx=minimal or
  - http://localhost:8080/?appCtx=projected

in a browser. 
```
What do you think?
```

Reply:
Was thinking of that, but wanted to keep the README compact (refer to dedicated docs later). But fine to add more options.